### PR TITLE
LIVE-2767 LLC - Countervalues API method change

### PR DESCRIPTION
**Pull request by LFBarreto**
### 📝 Description

_Updated LLC countervalues pair method from POST legacy format to GET format_

### ❓ Context

- **Impacted projects**: `ledger-live-common` affects -> `ledger-live-desktop`  `ledger-live-mobile`   
- **Linked resource(s)**: [LIVE-2767] 

### ✅ Checklist

- [X] **Test coverage** 
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

To test:
check on LLD that theres no regression in countervalues shown and that the switch of countervalues is correct.
basically the goall is to have the same values before this update than after.

The impacted areas are all countervalues related to accounts imported by the nano (the market page is not impacted) 
you can check it on those sections but limited to:
- Portfolio page
- Operation details
- Accounts pages
- Send Flows countervalues
- Swap flows countervalues

[LIVE-2767]: https://ledgerhq.atlassian.net/browse/LIVE-2767?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/ledger-live-common/src/countervalues/api/api.ts [hunk 1]
```diff
@@ -8,12 +8,15 @@ const baseURL = () => getEnv("LEDGER_COUNTERVALUES_API");
...
   const { data } = await network({
-    method: "POST",
-    url: `${baseURL()}/latest${direct ? "?method=direct" : ""}`,
-    data: pairs.map((p) => ({
-      from: p.from.countervalueTicker ?? p.from.ticker,
-      to: p.to.countervalueTicker ?? p.to.ticker,
-    })),
+    method: "GET",
+    url: `${baseURL()}/latest${direct ? "/direct" : "/indirect"}?pairs=${pairs
+      .map(
```
Comment (gre) [inline libs/ledger-live-common/src/countervalues/api/api.ts:13]: there don't seem to be any chunking logic here, but maybe it's somewhere else?
Can you confirm it won't break if there are like 1000 pairs to fetch?

**Comment by gre** [comment]
How is this test covered at the moment? It isn't unless you run the integration test https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/countervalues/logic.integration.test.ts

So please run the integration tests of countervalues

you can do this by going to https://github.com/LedgerHQ/ledger-live/actions/workflows/test-integration.yml and running a workflow on this branch and also using `countervalues` as a filter.

**Review by ychen-ledger — APPROVED** [review]
Validated with: 

- LLD 2.44.0-sha.6923cc610faadbe87416c36a1a9ebaa7c8de9907
- MacOS 
- Slightly difference cause of synchronization side as comment in jira https://ledgerhq.atlassian.net/browse/LIVE-2767
- LLM 3.3.0(66334)
-Android 12

**Comment by gre** [comment]
the integration tests are not testing edge cases. I have added a test and it shows api goes HTTP 414 with too much pairs asked.

**Review by gre — CHANGES_REQUESTED** [review]
https://github.com/LedgerHQ/ledger-live/pull/501 please take this in and recheck the QA 🙏

**Review by gre — APPROVED** [review]

**Comment by gre** [comment]
@LFBarreto do you know if this PR is in sync with develop? the codecov diff is strange but I wonder if it's not due to fact we recently removed codecov coming from the "integration". normally codecov is now only on unit test and should be more stable

**Comment by gre** [comment]
https://github.com/LedgerHQ/ledger-live/actions/runs/2697573130
